public/main.css

```diff
@@ -321,49 +868,103 @@ a {
 /* animations */
 
 .pulse {
+  -webkit-animation: pulse 0.5s linear infinite alternate;
   animation: pulse 0.5s linear infinite alternate;
 }
+
 .pop {
+  -webkit-animation: pop 0.3s;
   animation: pop 0.3s;
 }
+
 .fadein {
+  -webkit-animation: fadein 0.5s;
   animation: fadein 0.5s;
 }
 
+@-webkit-keyframes pulse {
```

Comment:
this autoprefixing probably isn't necessary in 2022 but I guess it doesn't hurt much